pylint/test/functional/arithmetic_operators.py

```diff
@@ -0,0 +1,8 @@
+# pylint: disable=missing-docstring,invalid-name,unused-argument
+def sample_function(*args, **kwargs):
```

Comment:
How does this behave to operator with different priorities? For instance, PEP 8 recommends something as ``x = x*2 - 1`` instead of ``x = x * 2 - 1``.